sunpy/net/dataretriever/client.py

```diff
@@ -41,12 +41,18 @@ def __init__(self, map0, url, time=None):
         self.time = TimeRange(map0.get('Time_start'),
                               map0.get('Time_end')) if time is None else time
         self.wave = map0.get('wavelength', np.NaN)
+        if meta is not None:
```

Comment:
What happens if meta is None?

Reply:
Results remain same as they were earlier, no change. If not None, only then it shows new info.